packages/components/Tabs/src/Tabs.tsx

```diff
@@ -10,7 +10,8 @@ import { settings } from './Tabs.settings';
 import { mergeSettings } from '@uifabricshared/foundation-settings';
 import { filterViewProps } from '@fluentui-react-native/adapters';
 import { foregroundColorTokens, textTokens, backgroundColorTokens } from '@fluentui-react-native/tokens';
-import { useSelectedKey } from '@fluentui-react-native/interactive-hooks';
+import { useSelectedKey, useAsPressable } from '@fluentui-react-native/interactive-hooks';
+import type { IKeyboardEvent } from '@office-iss/react-native-win32';
```

Comment:
Nit: importing something from react-native-win32 in the platform agnostic tabs file means we either don't get the import on other platforms, or now other platforms like macOS / windows depend on react-native-win32. If possible, I would move this to a `Tabs.win32.tsx` specific file. It looks like you're using the type for the event handler at line 94, which I think you might also be fine to leave as the `any` type (I've seen other event handlers do that for better or worse).